### PR TITLE
Adding terraform example with AWS resources

### PR DESCRIPTION
**Pull request by AvadheshKaria**
Description says it all, i.e. no other code change

### CODE REVIEW

File: website/docs/r/kinesis_metrics_source.html.markdown [hunk 1]
```diff
@@ -128,4 +128,282 @@ HTTP sources can be imported using the collector name and source name (`collecto
...
+  role_arn      = aws_iam_role.metric_stream_to_firehose.arn
+  firehose_arn  = aws_kinesis_firehose_delivery_stream.kinesis_stream.arn
+  output_format = "opentelemetry0.7"
+
+//  include_filter {
+//    namespace = "AWS/ApplicationELB"
+//  }
+//  include_filter {
+//    namespace = "AWS/DynamoDB"
+//  }
```
Comment (vsinghal13) [inline website/docs/r/kinesis_metrics_source.html.markdown:207]: does this need to be uncommented?

File: website/docs/r/kinesis_metrics_source.html.markdown [hunk 2]
```diff
@@ -128,4 +128,282 @@ HTTP sources can be imported using the collector name and source name (`collecto
...
+      for_each = local.tagfilters
+      content {
+        type      = tag_filters.value.type
+        namespace = tag_filters.value.namespace
+        tags      = tag_filters.value.tags
+      }
+    }
+  }
+}
+
```
Comment (vsinghal13) [inline website/docs/r/kinesis_metrics_source.html.markdown:193]: should we remove the sumologic resources as they are already mentioned above in the Example usage section?

File: website/docs/r/kinesis_metrics_source.html.markdown (hunk 1 above)
Comment (AvadheshKaria) [reply website/docs/r/kinesis_metrics_source.html.markdown:208]: I added comment to explain the lines, i.e. I think adding commented example is good in this case

File: website/docs/r/kinesis_metrics_source.html.markdown (hunk 2 above)
Comment (AvadheshKaria) [reply website/docs/r/kinesis_metrics_source.html.markdown:193]: I am thinking if someone just does copy and paste of the full section it should work.
I was working with customer, and I feel copy paste should work for one example end-to-end.

**Review by vsinghal13 — APPROVED** [review]

**Review by kagarwal06 — APPROVED** [review]

**Pull request merged**